src/switch/switch.component.ts

```diff
@@ -134,5 +141,9 @@ export class SwitchComponent extends CheckboxComponent implements OnInit {
 		this.renderer.addClass(labelEl, labelClass);
 		this.renderer.addClass(buttonEl, buttonClass);
 		*/
+		if (this.size === "sm") {
+			const inputEl = this.elementRef.nativeElement.querySelector("input");
+			this.renderer.addClass(inputEl, "bx--toggle--small");
+		}
```

Comment:
Could you instead remove the entire `ngOnInit` block and use an `[ngClass]` on the input?

Reply:
Sure. :)